assets/colors.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Do we want/need to merge this into a single JSON file? Perhaps even including the PNGs as base64 encoded?

Why?
1. Only one fetch from the client
2. Adding things to it doesn't require changes to the client

Why Not?
1. Parallel fetch is faster thanks to HTTP improvements
2. Easier to update to new versions for partial bits as the cache hit can skip other things

Perhaps just merging the json into one "settings.json" and while longer loop, it references the pngs (and perhaps svgs, etc...?) so that it is easier to have different images for different setups?